### DISCUSSION TITLE
Secret scanning patterns on Enterprise level

### DISCUSSION THREAD

**Discussion by tenjaa** [community/community#16520] [Enterprise] [answered]
We have some very common patterns for internal secrets.
They should be applied to every org in our enterprise.

Right now I can only view alerts on an enterprise level but not manage new patterns.

**Comment by just-joshing** [comment, marked as answer]
Hello! You should be able to create new patterns under the "Advanced Security" section of the "Policies" tab at the enterprise-level.

https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/defining-custom-patterns-for-secret-scanning#defining-a-custom-pattern-for-an-enterprise-account

**Reply by tenjaa** [reply]
Ok thats awesome, just overlooked it. Thanks!
